text/0000-sigstore-integration.md

```diff
@@ -0,0 +1,387 @@
+- Feature Name: cargo_crates_sigstore_integration
+- Start Date: 2023-03-27
+- RFC PR: [rust-lang/rfcs#0000](https://github.com/rust-lang/rfcs/pull/0000)
+- Rust Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
+
+Authors:
+
+* Tim Pletcher, Hewlett Packard Enterprise
+* Ulf Lilleengen, Red Hat / IBM
+
+# Summary
```

Comment:
I think this summary is far too long, and entirely too full of jargon[^1].

It may be worth looking at the summaries of some of the accepted RFCs, which are all much shorter and get to the point much more directly. (The guidance in the [template](https://github.com/rust-lang/rfcs/blob/master/0000-template.md) that it should be "one paragraph" is not a hard rule, but this is far too much)

[^1]: Look, I've worked in security-adjacent software for much of my career and had to look things up, so I'm unsure who the intended audience is.

Reply:
Seconding this: even the most jargon-filled academic abstracts remain a single paragraph.

The sole purpose of the summary is to explain what the RFC is. What is SigStore, what will the effect be for end users, etc.?

Anything else is just fluff. There's plenty of time to elaborate on motivation in the motivation section.

Reply:
Those are both good point, and I've given a first stab at simplifying it. We were concerned based on the pre-RFC discussions that we had to provide more background for this change, clearly it was too much for the summary :grimacing: